service/fuzz_test/Makefile.mk

```diff
@@ -0,0 +1,40 @@
+#  Copyright (c) 2015 - 2023, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+fuzz_test_dir = fuzz_test
```

Comment:
This parameter is used on lines 7-9 but not lines 23 - 25.  I'd be okay omitting the "fuzz_test_dir" variable, or using it throughout in place of "fuzz_test"